templates/repo/view_list.tmpl

```diff
@@ -28,6 +28,10 @@
 						{{end}}
 				</a>
 				<span class="grey has-emoji">{{RenderCommitMessage .LatestCommit.Summary .RepoLink $.Repository.ComposeMetas}}
+                {{if IsMultilineCommitMessage .LatestCommit.Message}}
```

Comment:
Please indent with tabs

Reply:
I'll fix it, Golands for some reason doesn't want to respect the settings.